turbo.json

```diff
@@ -236,6 +236,8 @@
     "BLACKLISTED_GUEST_EMAILS",
     "AUTH_BEARER_TOKEN_VERCEL",
     "BUILD_ID",
+    "CAL_SIGNATURE_TOKEN",
+    "CALCOM_PRIVATE_API_ROUTE",
```

Comment:
do we need to add these variables to the env.example file?

Reply:
Good spot! On it